docs/comparison.rst

```diff
@@ -0,0 +1,83 @@
+
+Fast Holographic Deconvolution (FHD)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+FHD is a data calibration and modeling framework written in IDL by researchers at the University of Washington for analyzing data from the Murchison Widefield Array (MWA) in Western Australia. It calculates sky models by convolving a high-resolution psf with the exact DFT of point sources. This is approach is more efficient when working with wide primary beams, which will be narrower in Fourier space and thus limit the size of the convolution to be performed.
+
+Advantages:
+
+1. Fast.
+2. Well-tested through years of use.
+3. Support for diffuse foreground models.
+4. Support for discrete pointings.
+5. Uses IDL's native parallelization efficiently.
+
+Disadvantages:
+
+1. Written in the proprietary IDL language.
+2. Designed for the MWA, and has limited support for other instruments.
+3. The discrete convolution in Fourier Space can introduce aliasing and ringing artifacts at a level relevant to 21cm cosmology.
+4. Measurement equation formalism relies on the flat-sky approximation, limiting curved sky effects.
+5. Primary beam motion on the sky is limited to the "snapshot" size, not the time step size.
+6. Not parallelized over frequency.
+
+.. image:: fhd_uvsim_compare.png
+    :width: 600
+    :alt: Delay-spectrum comparison of FHD and pyuvsim point source simulations.
+
+The figure above shows a comparison of a `pyuvsim` simulation of several point sources with a corresponding FHD simulation. The visibilities have been Fourier-transformed along the frequency axis (a "delay transform") with a Blackman-Harris window function, squared, and then averaged in time. This forms a so-called *delay spectrum*, which is proportional to the power spectrum modes along the line of sight. For smooth-spectrum foregrounds, the power at high delays should be very low. The FHD simulation (shown in orange) introduces excess power at high delays, which can be a real problem for 21cm cosmological research.
+
+Common Astronomy Software Applications (CASA)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CASA is a set of tools published by the National Radio Astronomical Observatory (NRAO), and has tools available for visibility simulation. Fully parallelized by partitioning Measurement Set (MS) files into smaller tasks and distributed with MPI across any number of available processing units. Non-trivial parallelization also available via OpenMP for shared memory computations on a single node.
+
+Advantages:
+
+1. Uses compiled C code internally, which is faster.
+2. Established in the field already and very well-documented.
+3. OpenMP utilizes shared memory on a single node if the calculation an be decomposed efficiently. MPI for all other parallel processing needs.
+4. Support for a source component lists and FITS image source models.
+
+Disadvantages:
+
+1. Limited support for user-defined primary beam models.
+2. Lacks full-sky coverage.
```

Comment:
Reword this. There are 2 potential issues:
1) the measurement equation it uses is almost certainly not correct for wide-field sources (Nithya will check on wording here)
2)The internal uvw rotation (for phasing) is known to be incorrect, this affects the coherence of sources far from the phase center.

Reply:
I spoke recently to some CASA developers. They claim they can achieve full sky coverage if the model and the beam (or model times custom beam) covers the full sky. They also claim to achieve it accurately without making flat-sky or narrow-field approximations by using the brute force DFT equation. So stating "lacks full-sky coverage" is not accurate. I don't know about the phasing issues though, which sounds like a different issue. If we want to insist on that point of phasing accuracy, perhaps we need to quote some reference or outstanding ticket that acknowledges that problem.

Reply:
@dannyjacobs can you get us a ticket number?

Reply:
@nithyanandan When you say "brute force DFT", does this mean they're not gridding the source model anymore in that case?

Reply:
(never mind, I just saw your previous message in the datasim channel)